memlab.bib

```diff
@@ -110,6 +110,27 @@ @string{vn
 @string{vr = {Vision Research}}
 
 
+@article{Gros88,
+	Author = {Grossberg, S.},
+	Date-Added = {2020-06-16 18:07:07 +0000},
+	Date-Modified = {2020-06-16 18:07:07 +0000},
+	Journal = {Neural networks},
+	Number = {1},
+	Pages = {17--61},
+	Title = {Nonlinear neural networks: Principles, mechanisms, and architectures.},
+	Volume = {1},
+	Year = {1988}}
+
+@article{Frist00a,
```

Comment:
Should be Fris00a (or Fris00 if there isn't another Friston article from 2000)

Reply:
there are 3 from that year, published under 'the labile brain' and in the same journal